index.html

```diff
@@ -0,0 +1,309 @@
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <div class="languages">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+                    <button>See Project</button>
+                </div>
+        </div>
+        <div class="item">
+                <img src="./images/mob_Portfolio4.png" alt="img1">
+                <div class="info_projet">
+                    <h2>Tonic</h2>
+                    <div class="type">
+                        <p class="client">CANOPY</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">Back End Dev</p>
+                        <img src="./images/circle.png" alt="">
+                        <p class="service">2015</p>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized
+                        reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="languages">
+                        <p>html</p>
+                        <p>css</p>
+                        <p>javaScript</p>
+                    </div>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- about section -->
+
+    <section class="about">
+        <div class="about_me">
+            <h2>About Myself</h2>
+            <p>
+                Hello I’m a software developer! I can help you
+                build a product , feature or website Look through
+                some of my work and experience! If you like what
+                you see and have a project you need coded, don’t 
+                hesitate to contact me.
+            </p>
+            <h3>LET'S CONNECT</h3>
+            <div class="social">
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-mastodon"></i></i></a>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            </div>
+            <button>Get my resume</button>
+        </div>
+        <div class="expertise">
+            <div class="domaine">
+                <h2>languages</h2>
+                <i class="fa-solid fa-chevron-down"></i>
+            </div>
+            <div class="list_language">
+                <div class="lang"><img src="./images/js.png" alt="js"><h3>javaScript</h3></div>
+                <div class="lang"><img src="./images/html.png" alt="js"><h3>HTML</h3></div>
+                <div class="lang"><img src="./images/css.png" alt="js"><h3>CSS</h3></div>
+            </div>
```

Comment:
Well done here 👍🏽   but the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list of `ul` and each of them a `li`. This will help browsers understand your page a bit better. 😉

Reply:
[Mandatory]

As mentioned by @didierganthier, better to use the ul and li tags in order to respect the semantics.

Please take [this link as a reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).